pandas/plotting/_core.py

```diff
@@ -889,6 +909,68 @@ def _make_plot(self):
             ax.errorbar(data[x].values, data[y].values,
                         linestyle='none', **err_kwds)
 
+    def _sci_notation(self, num):
+        """
+        Returns mantissa and exponent of the number passed in argument.
+        Example:
+        >>> _sci_notation(89278.8924)
+        (8.9, 5.0)
+        """
+        scientific_notation = '{:e}'.format(num)
+        expnt = float(re.search(r'e([+-]\d*)$',
```

Comment:
Hmm I was hoping there would be a better way to do this than using `re` but couldn't find anything better on a Google search myself as I'm sure you already tried ha. With that said, any way to do this in one search and return the appropriately matched groups instead of doing two passes?

Reply:
Indeed I did the same and could not find a better way.
I modified as you suggest to make only one regexp search.

Reply:
I'd also suggest putting it as a nested function where it is actually used (i.e. in `_legend_bubbles`). Even though it's private I don't think this should be an instance method